src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -2,13 +2,13 @@
   "issuerUrl": "http://docker.for.win.localhost:5102",
   "clients": [
     {
-      "clientId": "SampleClient",
-      "clientName": "Sample Client",
+      "clientId": "NodeClient",
+      "clientName": "Node Client",
       "allowedGrantTypes": "Code",
       "allowOfflineAccess": false,
       "requireClientSecret": true,
+      "secret": "NodeSecret",
```

Comment:
This might be a silly question, but is there any chance we can make it work with clientId remaining "SampleClient"
and the secret staying "SampleClientSecret"?

Having it as it stands in this PR would not work with k8s' default config and would be a pain to amend.